### ISSUE TITLE
Request for information.

### ISSUE THREAD

**Issue by NegassaB** [atlassian/commonmark-java#143]
I apologize for doing it like this, but I couldn't reach you any other way.

I wish to use your library in an application am trying/failing to create. I want to build a sticky note application that supports markdown. My question is can I use your library to do so? Ideally, the user will just type the necessary formatting with the text on to the sticky note and it will be saved. 

Does your library support such kind of operation?

Thank you.

**Comment by NegassaB** [comment]
No, I want to show it as markdown. Is this possible?

Negassa Berhanu
Freelance android & web developer
+251911985365

________________________________

**Comment by robinst** [comment]
The raw Markdown? Yeah that's possible, doesn't really have anything to do with this library, but yeah :).

**Comment by NegassaB** [comment]
Oh I must have misunderstood the library's purpose. All I want is the user to type in the notes using markdown syntax and that note to be displayed as markdown.

Thanks for your cooperation.

Negassa Berhanu
Freelance android & web developer
+251911985365

________________________________

**Comment by robinst** [comment]
Still not sure what exactly you want. If you're looking for rendering on Android, see https://github.com/noties/Markwon (which is based on commonmark-java).

**Comment by NegassaB** [comment]
I think I am doing a terrible job expressing what I want.
If you know boostnote, you'll know that it allows you to write markdown syntax and display it. That's what I want to build for my sticky note, so you'll be able to write markdown syntax and it will be displayed as such, but instead of using JS I am using java.
I hope I've cleared it up.

Negassa Berhanu
Freelance android & web developer
+251911985365

________________________________

**Comment by NegassaB** [comment]
And what you sent is perfect for the android version of my application, thank you

Negassa Berhanu
Freelance android & web developer
+251911985365

________________________________

**Comment by robinst** [comment]
> so you'll be able to write markdown syntax and it will be displayed as such

Yeah you can use this library for that. Depending on how you need to display it, you might have to write your own rendering code. This library comes with HTML rendering, but it's pretty simple to write your own renderer too.

**Comment by NegassaB** [comment]
Yea it becoming apparent that I might have to do that.
Thanks for all you help. I am going to study your code to understand and build my own renderer.

Thank you very much.

Negassa Berhanu
Freelance android & web developer
+251911985365

________________________________
